packages/lesswrong/lib/eaGivingSeason.ts

```diff
@@ -0,0 +1,55 @@
+export const eaGivingSeason23ElectionName = "givingSeason23";
```

Comment:
Everything in this file is just refactored from other files to try to keep all the constants in a single place.